Makefile.am

```diff
@@ -225,6 +225,40 @@ EXTRA_DIST += \
   javascript/ola/port_table.js \
   javascript/ola/universe_control.js \
   javascript/ola/universe_item.js \
+  javascript/new-src/README.md \
+  javascript/new-src/bower.json \
+  javascript/new-src/src/js/app.js \
```

Comment:
everything in olad/www/new is already referenced, so every reference  to javascript/new-src/src can be deleted?